### ISSUE TITLE
I Want to ask about your old code

### ISSUE THREAD

**Issue by cyun-404** [Lornatang/ESRGAN-PyTorch#14]
hello~
I'm using your last code(befor update new code). that code didnt use sigmo but torch.full, It make diffrent result image??

I wonder if it would be better to experiment with your new code to check the esrgan results. Is there a big difference in the results?

**Comment by Lornatang** [comment]
Which line of code are you referring to?

**Comment by cyun-404** [comment]
In [/train_esrgan.py] 283 line.
I wonder if there is a big difference between bcewithlogitsloss and sigmoid. 
Is there a reason you used bcewithlogitsloss in your previous code and changed it to sigmoid?

**Comment by Lornatang** [comment]
Because BCELoss training is not stable enough, BCEWithlogits is relatively more stable, which is prone to model collapse.

**Comment by cyun-404** [comment]
thank you!!
